### PR TITLE
Added support for replication channels under MySQL 5.7 (display M)

### PR DESCRIPTION
**Pull request by sjalilund**

### CODE REVIEW

File: innotop [hunk 1]
```diff
@@ -9740,22 +9758,56 @@ sub get_master_slave_status {
...
+               		(($res->{slave_sql_running} || 'Yes') eq 'Yes'
+               		&& ($res->{slave_io_running} || 'Yes') eq 'Yes') ? 'Yes' : 'No';
+         	}
+         	else {
+            	$vars->{Slave_ok} = 'Off';
+         	}
+	  	 } else {
+				 my $dbh = connect_to_db($cxn);
+				 my $sql = 'SHOW SLAVE STATUS FOR CHANNEL \'' . $channel . '\''; 
+				 #my $stmt = $dbh->prepare("call DbaTools.SP_Slave_Status('$channel')" ) ;
```
Comment (grypyrg) [inline innotop]: maybe remove this line, as it looks specific to your environment.

**Comment by grypyrg** [comment]
@sjalilund it works when you have multi source replication:

```
[RO] Replication Status (? for help)       127.0.0.1, 9m, 8.16 QPS, 5/1/0 con/run/cac thds, 5.7.11-4-log

___________________________________________ Slave SQL Status ___________________________________________
Channel  Master     Master UUID                           On?  TimeLag  Catchup  Temp  Relay Pos  Last E
one      localhost  d7e93be0-0459-11e6-a7b2-08002774c31b  Yes    00:00     0.00              327        
two      localhost  5b9d58e4-045a-11e6-b072-08002774c31b  Yes    00:00     0.00                4        

___________________________________________ Slave I/O Status ___________________________________________
Channel  Master     Master UUID                           On?  File                     Relay Size  Pos 
two      localhost  5b9d58e4-045a-11e6-b072-08002774c31b  No   57-community-bin.000003         154  922 
one      localhost  d7e93be0-0459-11e6-a7b2-08002774c31b  Yes  57-community-bin.000003         545  678 

____________________________________________ Master Status _____________________________________________
File                     Position  Binlog Cache  Executed GTID Set  Server UUID                         
57-community-bin.000003       154         0.00%  N/A                b40426f3-045a-11e6-bbe4-08002774c31b

```

BUT, when I have regular replication with no channels, I don't have anything anymore:
```
[RO] Replication Status (? for help)     127.0.0.1, 14m, 19.18 QPS, 4/1/0 con/run/cac thds, 5.7.11-4-log

___________________________________________ Slave SQL Status ___________________________________________
Channel  Master  Master UUID  On?  TimeLag  Catchup  Temp  Relay Pos  Last Error  Retrieved GTID Set  Ex

______________________ Slave I/O Status _______________________
Channel  Master  Master UUID  On?  File  Relay Size  Pos  State

____________________________________________ Master Status _____________________________________________
File                     Position  Binlog Cache  Executed GTID Set  Server UUID                         
57-community-bin.000002       922         0.00%  N/A                5b9d58e4-045a-11e6-b072-08002774c31b

```

**Comment by grypyrg** [comment]
@sjalilund: Thanks, seems to work now.

@lefred: can you test and 👍  this?

**Comment by lefred** [comment]
for info, I'm implementing test for this. If it's OK, I'll merge it. Thank you for your contribution

**Pull request merged**